jwst/ramp_fitting/gls_fit.py

```diff
@@ -576,7 +576,7 @@ def gls_fit(ramp_data, input_var_data,
         pixel, in electrons per second.
 
     readnoise: 1-D ndarray, length nz.
-        The read noise in electrons at each detector pixel.
+        The read noise in electrons at each deector pixel.
```

Comment:
Was the creation of a typo intentional here?

Reply:
I removed the typo